### 10.1 抛出异常

### 10.2 取得反向跟踪的字符串
如果Python 遇到错误，它就会生成一些错误信息，称为“反向跟踪”。<br>
反向跟踪包含了出错消息、导致该错误的代码行号，以及导致该错误的函数调用的序列。<br>
这个序列称为“调用栈”。<br>


